README.md

```diff
@@ -65,6 +65,31 @@ you will be able to see the Results objects of the analysis after some minutes (
         "details": "The error message means that the service in Kubernetes doesn't have any associated endpoints, which should have been labeled with \"control-plane=controller-manager\". \n\nTo solve this issue, you need to add the \"control-plane=controller-manager\" label to the endpoint that matches the service. Once the endpoint is labeled correctly, Kubernetes can associate it with the service, and the error should be resolved.",
 ```
 
+## LocalAI Example
+
+1. Install the operator from the [Installation](#installation) section.
+
+2. Follow the [LocalAI installation guide](https://github.com/go-skynet/helm-charts#readme) to install LocalAI. No OpenAI secret is required when using LocalAI!
```

Comment:
Can we remove the exclamation mark? 

Maybe use italics (_No OpenAI secret is required when using LocalAI_)